packages/jellyfish-api-core/src/category/mining.ts

```diff
@@ -6,6 +6,15 @@ export enum EstimateMode {
   CONSERVATIVE = 'CONSERVATIVE'
 }
 
+export enum MasternodeState {
+  PRE_ENABLED = 'PRE_ENABLED',
+  ENABLED = 'ENABLED',
+  PRE_RESIGNED = 'PRE_RESIGNED',
+  RESIGNED = 'RESIGNED',
+  UNKNOWN = 'UNKNOWN',
+  TRANSFERRING = 'TRANSFERRING'
+}
```

Comment:
can this just import from `category/masternode.ts`, it seems to be identical 🤔

Reply:
So should masternode import from mining or mining import from masternode?
Every enum should be redeclared if there are in a different package

Reply:
At a glance I would say `MasternodeState` would be expected to come from the masternode category. All things masternode contained within its category. But if `MasternodeState` in mining is actually `MiningMasternodeState` then I would say they should be separated.

It seems sort of disjointed if our core is acting against different types of Masternodes but that may just be my own unfamiliarity with the overall design of core.